server/tests/routes/user.test.js

```diff
@@ -0,0 +1,146 @@
+import request from 'supertest-as-promised';
+import httpStatus from 'http-status';
+import chai, { expect } from 'chai';
+import app from '../../../index';
+import userService from '../../services/user';
+
+chai.config.includeStack = true;
+
+describe('## User APIs', () => {
+  let user = {
+    name: 'Jarryd Lee',
+    email: 'abc@google.com'
+  };
+
+  const invalidUser = {
+    name: 'Jarryd Lee',
+    email: 0
+  };
+
+  describe('# POST /api/v1/user', () => {
```

Comment:
The variables up above should be pulled inside here and this should be a context instead of a describe. Something like:

```
describe('POST') {
  context('to a valid user') {
    let user = {}
    it should...
  }
  context('with invalid user') {
    let invalidUser = {}
    it should...
  }
}
```